core/src/main/java/org/arquillian/smart/testing/SystemPropertyResolver.java

```diff
@@ -0,0 +1,33 @@
+package org.arquillian.smart.testing;
+
+public class SystemPropertyResolver {
```

Comment:
It is called SystemPropertyResolver but it just resolves git properties, so maybe another name such as Git `SystemPropertiesResolver` ? It is just an idea I am really bad giving names to things.